spec/requests/service_templates_spec.rb

```diff
@@ -604,33 +604,54 @@
       end
     end
 
-    context "with an unorderable template" do
```

Comment:
NOTE I created two contexts here because the old context said it was unorderable but then expected it to be orderable but not displayed.

I created a context where it does not have the orderable trait and expect the `Service template does not belong to a service catalog` failure, then a second where it is orderable but not displayed and expect the `Service template is not configured to be displayed` error (which was the original expectation).